<ui version="4.0" >
 <class>BTSettings</class>
 <widget class="QWidget" name="BTSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>152</width>
    <height>177</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Bluetooth</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QListWidget" name="actionSelection" >
     <property name="currentRow" >
      <number>-1</number>
     </property>
     <item>
      <property name="text" >
       <string>Device</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Visibility</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Device Info</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Local Services</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Remote Devices</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>My Headset</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>My Handsfree</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
